# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1215725780 14400
#      Thu Jul 10 17:36:20 2008 -0400
# Node ID fcc70f30fe31cd4bc694ce773df17bb69131f8c1
# Parent  e56bb65186f63d4bf82f8a9308fbef45548df372
configure.in: use #endif, not #fi in test

diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -660,17 +660,17 @@ if test "x$OPENGL_LIBS" != "x"; then
 	save_CXXFLAGS="$CXXFLAGS"
 	LIBS="$LIBS $OPENGL_LIBS $FT2_LIBS -lftgl"
 	CXXFLAGS="$CXXFLAGS $FT2_CFLAGS"
         AC_TRY_LINK([
 #ifdef HAVE_FTGL_FTGL_H
 #include <FTGL/ftgl.h>
 #elif HAVE_FTGL_H
 #include <ftgl.h>
-#fi], [
+#endif], [
 FTGLTextureFont font("");], [
           OPENGL_LIBS="$OPENGL_LIBS $FT2_LIBS -lftgl"
 	  LIBS="$save_LIBS"
           AC_DEFINE(HAVE_FTGL, 1, [Define to 1 if FTGL is present])
 	  AC_MSG_RESULT(yes)
 	  XTRA_CXXFLAGS="$XTRA_CXXFLAGS $FT2_CFLAGS"],[
 	  AC_MSG_RESULT(no)])
 	LIBS="$save_LIBS"
